MockPyth.sol

```diff
@@ -17,9 +17,10 @@ contract MockPyth is AbstractPyth {
     function updatePriceFeeds(bytes[] memory updateData) public override {
         for(uint i = 0; i < updateData.length; i++) {
             PythStructs.PriceFeed memory priceFeed = abi.decode(updateData[i], (PythStructs.PriceFeed));
+            emit PriceUpdate(priceFeed.id, true, priceFeed.publishTime, priceFeeds[priceFeed.id].publishTime, priceFeed.price, priceFeed.conf);
             priceFeeds[priceFeed.id] = priceFeed;
-            emit PriceUpdate(priceFeed.id, priceFeed.publishTime);
         }
+        emit UpdatePriceFeedsCall(msg.sender, 0, 0, updateData.length, updateData.length);
```

Comment:
Is there a reason this doesn't implement the same logic as the actual contract for the updated flag and such?

please document any discrepancies between the emitted events here and by the real contract.

Reply:
Well simply because it's a mock and abstracts away wormhole. Just gets serialized price feed array. 
So there is no chain_id and sequence_id. 

However I'll add a check for timestamp to ensure it updates only newer messages.

Reply:
Added sequence number as well.